### PR TITLE
add docs for daily merge

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: site/content/internal/release-process.md [hunk 1]
```diff
@@ -43,4 +43,39 @@ After a couple days pass you will need to set the CI servers to point back to `m
 1. Post `/mb setci master` into a private channel.
 2. Post `/mb setprerelease master` into a private channel.
 
+# Daily Merges from Release Branch to Master Branch
+
+During the release/rc cuts we need to merge daily the release branch into master to sync all changes made in the release branch.
```
Comment (jwilander) [inline site/content/internal/release-process.md]: Maybe we should add a note about asking core committers to hold off merging?

Comment (cpanato) [reply site/content/internal/release-process.md]: sure thing! thanks for the review!

**Comment by cpanato** [comment]
so we merge this and the other one? or close this? @jasonblais

File: site/content/internal/release-process.md (hunk 1 above)
Comment (cpanato) [reply site/content/internal/release-process.md]: done

**Comment by cpanato** [comment]
please @jasonblais @jwilander PTAL

**Comment by jasonblais** [comment]
I merged the tweaks from https://github.com/mattermost/mattermost-developer-documentation/pull/121, hope that was okay.

**Pull request merged**